### PR TITLE
[OWL-402] add script for create event table

### PR DESCRIPTION
**Pull request by masato25**

### CODE REVIEW

File: dbpatch/change-log/schema-portal/masato-1.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+SET NAMES 'utf8';
+
+DROP TABLE IF EXISTS event;
```
Comment (mikelue) [inline dbpatch/change-log/schema-portal/masato-1.sql]: Remove this line

File: dbpatch/change-log/schema-portal/masato-1.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+CREATE TABLE event
```
Comment (mikelue) [inline dbpatch/change-log/schema-portal/masato-1.sql]: Missed `IF NOT EXISTS`

File: dbpatch/change-log/change-log-portal.yaml [hunk 3]
```diff
@@ -18,3 +18,8 @@
     filename: "mike-4.sql",
     comment: "Add name of agent"
 }
+- {
+    id: "masato-1",
```
Comment (mikelue) [inline dbpatch/change-log/change-log-portal.yaml]: Don't have to numbering by author, just by last number(whoever wrote it)

**Comment by hitripod** [comment]
Thank @mikelue

**Pull request merged**